### DISCUSSION TITLE
How can I pass env variables from within DSA to a image that is being added through slicer web task

### DISCUSSION THREAD

**Discussion by lomshabhishek** [andreped/FP-dsa-plugin#11] [Q&A]
I want to pass secret env variables when a new docker image is pulled and as well as eventually when it is used to create container to serve user request.

**Comment by andreped** [comment]
This sounds like something you should ask DSA about. Outside the scope of this plugin.

I see that you have made a separate Discussion in the DSA repo (see [here](https://github.com/DigitalSlideArchive/digital_slide_archive/discussions/300)). In the future, only make a single Discussion/issue at the appropriate repo.
